R/lazyframe__lazy.R

```diff
@@ -265,15 +265,71 @@ LazyFrame_filter = "use_extendr_wrapper"
 
 #' @title New DataFrame from LazyFrame_object$collect()
 #' @description collect DataFrame by lazy query
+#' @param type_coercion Boolean. Do type coercion optimization.
+#' @param predicate_pushdown  Boolean. Do predicate pushdown optimization.
+#' @param projection_pushdown  Boolean. Do projection pushdown optimization.
+#' @param simplify_expression  Boolean. Run simplify expressions optimization.
+#' @param no_optimization  Boolean. Turn off (certain) optimizations.
+#' @param slice_pushdown  Boolean. Slice pushdown optimization.
```

Comment:
Can you improve the description of the parameters? For example, change "Do type coercion optimization." to "Coerce types such that operations succeed and run on minimal required memory."

There are good descriptions of each param here: https://pola-rs.github.io/polars-book/user-guide/lazy/optimizations/